qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -95,6 +95,8 @@ def __init__(
              the local testing mode will be used, and primitive queries will run on a local simulator.
              For more details, check the `Qiskit Runtime local testing mode
              <https://docs.quantum.ibm.com/guides/local-testing-mode>`_ documentation.
+             The ``ibm_quantum`` channel is deprecated. For more details, read our
+             `migration guide <https://ibm.biz/classic-iqp-to-cloud-iqp>__`.
```

Comment:
```suggestion
             `migration guide <https://quantum.cloud.ibm.com/docs/migration-guides/classic-iqp-to-cloud-iqp>__`.
```

Reply:
I would use the real link so you never have to maintain it, but if you want to use the ibm.biz link, it should not point to the /en directory.  Just delete it and it will open in the correct language (once we have multiple languages).